src/Workspaces/Remote/ServiceHub/Shared/RoslynJsonConverter.SolutionIdConverters.cs

```diff
@@ -10,8 +10,13 @@ internal partial class AggregateJsonConverter : JsonConverter
     {
         private abstract class WorkspaceIdJsonConverter<T> : BaseJsonConverter<T>
         {
-            protected (Guid, string) ReadFromJsonObject(JsonReader reader)
+            protected (Guid, string)? ReadFromJsonObject(JsonReader reader)
             {
+                if (reader.TokenType == JsonToken.Null)
```

Comment:
These are needed because AddImport passes around null Ids (this hasn't happened before).